core/src/test/java/org/apache/iceberg/jdbc/TestJdbcCatalog.java

```diff
@@ -82,9 +81,8 @@
   private static JdbcCatalog catalog;
   private static String warehouseLocation;
 
-  @Rule
-  public TemporaryFolder temp = new TemporaryFolder();
-  File tableDir = null;
+  @TempDir
+  java.nio.file.Path tableDir;
```

Comment:
Import?

Reply:
required as otherwise it would clash with the `org.apache.hadoop.fs.Path` import

Reply:
Argh, okay - i looked through the imports but haven't seen that one.